path_keys_trim.go

```diff
@@ -56,10 +61,25 @@ Name of the key in Vault.
 
 		ExistenceCheck: b.pathKeysExistenceCheck,
 
-		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.CreateOperation: withFieldValidator(b.pathKeysTrimWrite),
-			logical.UpdateOperation: withFieldValidator(b.pathKeysTrimWrite),
-			logical.DeleteOperation: withFieldValidator(b.pathKeysTrimWrite),
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.CreateOperation: &framework.PathOperation{
+				Callback: withFieldValidator(b.pathKeysTrimWrite),
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationSuffix: "key-versions",
+				},
+			},
+			logical.UpdateOperation: &framework.PathOperation{
+				Callback: withFieldValidator(b.pathKeysTrimWrite),
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationSuffix: "key-versions",
+				},
+			},
+			logical.DeleteOperation: &framework.PathOperation{
+				Callback: withFieldValidator(b.pathKeysTrimWrite),
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationSuffix: "key-versions2",
```

Comment:
Does this need to have `2` appended? I don't see the duplicate

Reply:
These two are colliding:

| Path | Method | OperationID |
| ---- | ------ | ----------- |
| "/{gcpkms_mount_path}/keys/trim/{key}" | "post" | "google-cloud-kms-trim-key-versions" |
| "/{gcpkms_mount_path}/keys/trim/{key}" | "delete" | "google-cloud-kms-trim-key-versions2" |